client/changelist/change.go

```diff
@@ -0,0 +1,73 @@
+package changelist
+
+import (
+	"encoding/json"
+
+	"github.com/endophage/gotuf/data"
+)
+
+type tufChange struct {
+	Actn       string `json:"action"`
+	Role       string `json:"role"`
+	ChangeType string `json:"type"`
+	ChangePath string `json:"path"`
+	Data       []byte `json:"data"`
+}
+
+func NewTufChange(action, role, changeType, changePath string, content []byte) *tufChange {
+	return &tufChange{
+		Actn:       action,
+		Role:       role,
+		ChangeType: changeType,
+		ChangePath: changePath,
+		Data:       content,
+	}
+}
+
+func NewTufTargetChange(action, target string, hash map[string]data.HexBytes, size int64, custom map[string]interface{}) (*tufChange, error) {
```

Comment:
Every TUF target has a custom field that can contain any information another application wants to put in there. It's defined in the TUF spec and the only way to deal with it in Go is with a `map[string]interface{}`